packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -111,7 +111,7 @@ export function computeResourceEntryDetails(entry: RumPerformanceResourceTiming)
   }
 
   // Make sure a worker processing time is recorded
-  if (workerStart < fetchStart) {
+  if (0 < workerStart && workerStart < fetchStart) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** I generally find more readable when the variables precede static values ‏
```suggestion
  if (workerStart > 0 && workerStart < fetchStart) {
```

Reply:
I generally agree, except when we check whether a value is between two other values `a < b && b < c`